ucrop/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
```

Comment:
Yes, they are exception messages. It's not common to translate this kind of messages, because they're used for logging, not for showing to the user. Everything in the logcat is in english: having translated messages in the logs might be confusing. Also you can see that they're not translated in NL and SK.

I think these strings should not be externalized at all. They should be directly found in the source code.